pkg/config/loader_test.go

```diff
@@ -0,0 +1,330 @@
+package config
+
+import (
+	"os"
+	"reflect"
+	"testing"
+	"time"
+
+	autoheal "github.com/openshift/autoheal/pkg/apis/autoheal/v1alpha2"
+	batch "k8s.io/api/batch/v1"
+	core "k8s.io/api/core/v1"
+	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestFiles(t *testing.T) {
+	l := NewLoader()
+
+	filename0 := "test_file0.yaml"
+	filename1 := "test_file1.yaml"
+
+	data0 := `
+  awx:
```

Comment:
I think this can look even nicer if you indent the YAML text with tabs instead of spaces, that way it will align correctly wit the Go code when displayed using the standard 8 spaces per tab.